app/views/layouts/apipie_override.html.erb

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title><%= t('apipie.api_documentation', :default => 'Api Documentation') %></title>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <%= Apipie.include_stylesheets %>
+  <link type='text/css' rel='stylesheet' href='<%= Apipie.full_url("stylesheets/application.css")%>'/>
+  <!-- IE6-8 support of HTML5 elements -->
+  <!--[if lt IE 9]>
+  <script src="//html5shim.googlecode.com/svn/trunk/html5.js"></script>
+  <![endif]-->
+</head>
+<body>
+<div class="container-fluid">
+  <div class="row-fluid">
+    <div id='container'>
+      <%= yield %>
+      <%= render 'disqus' if Apipie.configuration.use_disqus? %>
+    </div>
+  </div>
+  <hr>
+  <footer><%= yield :apipie_footer %></footer>
+</div>
+<script type='text/javascript' src='/apidocs/javascripts/bundled/jquery.js'></script>
```

Comment:
Do we need this line since jquery is in yarn

Reply:
Good catch, I managed to replace the wrong line! 🤦